.github/workflows/lock.yml

```diff
@@ -0,0 +1,24 @@
+name: Lock closed issues and PRs
```

Comment:
All other workflows in this repo are generated through Projen – but (as the build succeeds) it does not seem to be a problem to mix files handled by Projen with manually written ones.
Alternatively, we could do it like in the AWS Adapter ([`/projen`](https://github.com/hashicorp/cdktf-aws-cdk/blob/main/projen)) and manage this file through Projen, too.
I'm fine with ignoring this and keeping it simple for now, though 😄

Reply:
That's a good point, I'd overlooked that one! In theory this should be fairly easy to do since we could just reuse `src/lock-issues.ts` which has essentially the same config. What would be the easiest, most lightweight way to do that?

Reply:
Probably set the `projenrcTs` option to `true` (like [here](https://github.com/hashicorp/cdktf-aws-cdk/blob/main/.projenrc.ts#L8)) (if we want to use TypeScript) and also possibly subclassing the current `JsiiProject` to add our workflow in the constructor (like we do in the AwsAdapter, too).

Reply:
I held off on the latter because it felt like overkill for adding a single workflow, but I think I managed to get everything else working.